src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -9,13 +9,13 @@ Use the following information to learn about backward incompatible changes that
 
 Some functionality previously included in the `{{site.data.var.ct}}` package is now provided in separate packages. These packages are composer dependencies for `{{site.data.var.ct}}`, which are installed and updated automatically when you install or update {{site.data.var.ct}}.
 
-The new architecture should not affect your install or update processes.  However, you might need to change some command syntax and processes when working with your {{site.data.var.ece}} project. For details, review the backward incompatible changes information and release notes for each package.
+The new architecture should not affect your install or update processes.  However, you might need to change some command syntax and processes when working with your {{site.data.var.ece}} project. For details, review the release notes and backward incompatible changes information for each package.
 
--  **`{{site.data.var.mcp}}` package**–See [Magento Cloud Patches changes](#magento-cloud-patches-changes) and [Release notes for magento/magento-cloud-patches]({{site.baseurl}}/cloud/release-notes/mcp-release-notes.html).
+-  **`{{site.data.var.mcp}}`**–See [Release notes for magento/magento-cloud-patches]({{site.baseurl}}/cloud/release-notes/mcp-release-notes.html) and [Magento Cloud Patches changes](#magento-cloud-patches-changes).
```

Comment:
I would do away with this entire list. Instead, I would provide a link in the sentence above and send them to the Cloud Suite landing page which should list all this information for each package neatly.